flutter/lib/src/sentry_flutter.dart

```diff
@@ -179,6 +185,12 @@ mixin SentryFlutter {
     // That allow us to send events to the network and then the Flutter integrations.
     final native = _native;
     if (native != null) {
+      // This is an Integration because we want to execute it after all the
+      // error handlers are in place. Calling a MethodChannel might result
+      // in errors. We also need to call this before the native sdk integrations
+      // so release is properly propagated.
+      integrations.add(LoadReleaseIntegration());
```

Comment:
have to move the release integration before the native sdk integration, does this have any consequences?

Reply:
Should be fine as long it is after the error integrations. Internally we only use package_info.